config-templates/redhat/local-volume-block/changelog.json

```diff
@@ -65,7 +81,7 @@
     },
     {
         "4.7": {
-            "status": "deprecated",
+            "status": "unsupported",
```

Comment:
When template is unsupported, we can remove `latest-revision` & `previous-revision` fields
Refer this https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/netapp/netapp-ontap-san/changelog.json#L49